src/app.js

```diff
@@ -1,11 +1,24 @@
 import express from "express"
-import {config} from "./config/app_config.js";
-import {client} from "./client/meteor_nasa_client.js";
+import {config} from "./config/config.js";
+import {client} from "./client/nasa_client.js";
 
 
 export const app = express();
 
-app.get('/meteors', client);
+app.get('/meteors', (req, res) => {
+    let startDate = req.query.start_date;
+    let endDate = req.query.end_date;
+
+    if (!startDate) {
+        startDate = new Date().toISOString().split('T')[0];
+    }
+
+    if (!endDate) {
+        endDate = new Date().toISOString().split('T')[0];
+    }
+
+    client(req, res, startDate, endDate);
```

Comment:
Name is not very informative. It's ok for right now, but it can become a trouble later